### ISSUE TITLE
[SF][A11y][ObjectStatus & Table]: Ensure that the contrast ratio for all text content is at least 4.5:1 in normal, hover, and keyboard focus state.

### ISSUE THREAD

**Issue by VidhiNPatel** [UI5/webcomponents-react#7929]
### Describe the bug

When using the <ObjectStatus> component within the <Table> for row actions, the contrast ratio of the ObjectStatus text content does not meet the minimum required ratio of 4.5:1.

This issue has happened for the sap_horizon normal and high contrast black and white theme as well.

### Isolated Example

https://stackblitz.com/edit/github-rztzxgrd-ygjsmrke?file=src%2FApp.tsx,package.json,index.html

### Expected Behaviour

Ensure that the contrast ratio for all text content is at least 4.5:1 in normal, hover, and keyboard focus state.

### UI5 Web Components for React Version

2.16.1

### UI5 Web Components Version

2.16

### Browser

Chrome

### Relevant log output

```shell

```

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
Hi @VidhiNPatel 

After reviewing the internal specifications (Wiki and Figma), the design guidelines currently define special color handling for ObjectStatus only when used in interactive lists or tables with rows in active state. For other states, the default component colors apply as intended.

For the `AnalyticalTable` component (developed in this repository), we'll implement the active state styling out of the box, so no additional configuration will be required.

For UI5 Web Components (`List`, `Table`, etc.), you'll need to apply custom CSS to override the text and icon colors when rows are in active state. To make this easier, we're:
  1. Simplifying icon color customization via CSS custom properties
  2. Adding documentation with examples showing how to proper set the active styles

**Comment by VidhiNPatel** [comment]
This issue is not fixed. I still can see the issue.  Can you please help to check and re-open this ?

**Comment by VidhiNPatel** [comment]
This issue is not fixed. I still can see the issue. Can you please help to check and re-open this ?

**Comment by Lukas742** [comment]
Hi @VidhiNPatel 

please see my comment above. In case you still face issues, add a reproducible example of the problem and we can take a look.

**Comment by VidhiNPatel** [comment]
Hi @Lukas742 ,

I have attached the example link:https://stackblitz.com/edit/github-rztzxgrd-ygsgchrr?file=src%2FApp.tsx,package.json,index.html  . For row actions, you may notice that the contrast ratio of the ObjectStatus text content does not meet the minimum required ratio of 4.5:1. This issue also occurs with the sap_horizon theme in both normal and high contrast black-and-white settings.

**Comment by Lukas742** [comment]
I checked again, and all ObjectStatus are implemented according to design and furthermore meet the 4.5:1 ratio. Additionally you didn't set the required CSS which is why it doesn't meet the criteria in active state.
